addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -880,33 +929,37 @@ export const editorCommands = {
         const [table] = editorCommands.insertHTML(editor, tableHtml);
         setCursorStart(table.querySelector('td'));
     },
-    addColumnLeft: editor => {
-        addColumn(editor, 'before');
+    addColumnLeft: (editor, referenceColumn) => {
+        return addColumn(editor, 'before', referenceColumn);
```

Comment:
Wait, come on. We have `_onTableAddColumnLeftClick` on editor, which calls `addColumnLeft` which is here, which itself calls `addColumn` ... so there are 3 levels of indirection between the handler and the actual function that is called. Those indirect functions do not even add any logic, they just forward the call. You could just as well have inlined `addColumn(editor, 'before', this._columnUiTarget)` where we do `addEventListener` and not add any indirection at all. This would remove like 8+ methods.

Reply:
Ok I inlined them at `addEventListener` but still calling `execCommand` (important for history management etc., that one does do a lot more than just redirecting). It would be nice to get rid of all these commands and just have `addColumn` instead but I don't believe it's possible to pass an argument to the command via the ~~stupid~~ XML API right? In which case it would break this:
![image](https://user-images.githubusercontent.com/24205914/186173353-01a5affb-71e1-4688-8cb9-11a4dfb3a06c.png)
I could modify that so execCommand fetches arguments from the XML but I think that's even more out of scope than all these other things I did in this PR :grimacing: